step1_SMS-PM/make/makefile

```diff
@@ -0,0 +1,121 @@
+ifeq ($(MESA_DIR),)
+ifeq ($($MESA_DIR_INTENTIONALLY_EMPTY),)
+  $(error MESA_DIR enviroment variable is not set)
+endif
+endif
+
+# copied the content from work_standard_makefile
+
+# change this as necessary.  see utils/makefile_header for definitions.
+
+include $(MESA_DIR)/utils/makefile_header
+
+LOAD_MESA = $(LOAD_EXTRAS1) $(LOAD_MESA_STAR) $(LOAD_EXTRAS)
+
+#################################################################
+
+ifndef STAR
+STAR = star
+endif
+
+# STAR_WORK_OBJS defined in makefile_header
+# added more code located in WORK_SRC_DIR
+IONIZATION_OBJS = ionization_def.o ion_tables_load.o mod_tables.o mod_ionization.o ion_tables_eval.o ion_table_plot.o ionization_lib.o
+INSPIRAL_OBJS = dop853_module.o CE_orbit.o CE_energy.o CE_torque.o CE_adjust_mdot.o CE_before_struct_burn_mix.o CE_after_struct_burn_mix.o CE_timestep.o CE_run_star_extras.o
```

Comment:
Is there a reason, why you compile the CE_inspiral code for step1, where none of this should be used?